Annotato/Annotato/Persistence/PersistenceService/OfflinePersistenceService/OfflinePersistenceService+AnnotationsPersistence.swift

```diff
@@ -17,6 +17,10 @@ extension OfflinePersistenceService: AnnotationsPersistence {
         return await localPersistence.annotations.deleteAnnotation(annotation: annotation)
     }
 
+    func createOrUpdateAnnotation(annotation: Annotation) -> Annotation? {
```

Comment:
In this layer, we set the timestamps for the other methods but we don't do that for `createOrUpdateX` functions. Is that okay?